product_docs/docs/epas/17/reference/oracle_compatibility_reference/epas_compat_bip_guide/03_built-in_packages/18_dbms_xmldom/getnodevalue.mdx

```diff
@@ -2,7 +2,7 @@
 title: "GETNODEVALUE"
 ---
 
-The `GETNODEVALUE` function provides the value of the node depending on its type.
+The `GETNODEVALUE` function provides the value of the node, depending on its type.
```

Comment:
Same query here.

Reply:
@sravanvelag Does `GETNODEVALUE` have the same behavior as `GETNODENAME`? Meaning that it supports DOMElement but for DOMText and DOMDocument it only provides a placeholder?

Reply:
It provides a value given a DOMText node and NULL given the other types. A change similar to what's proposed for GETNODENAME makes sense.

**The `GETNODEVALUE` function provides the value of the node**